README.md

```diff
@@ -14,7 +14,7 @@ The current state of this project is in the middle of "make it right".
 To run the crawler you'll need to do the following steps in the environment
 in which you want it to run:
 
- * Install Python 2.7.5
```

Comment:
Don't remember where this came from. Just curious, did version 2.7.5 not work in your environment?